alacritty_terminal/src/term/text_run.rs

```diff
@@ -0,0 +1,309 @@
+use super::{
```

Comment:
I'm not sure that `term/text_run.rs` is the best place for the text run. Sure, it's related to the terminal in some way, but it's also related to font and rendering.

I think the most uncontroversial thing would be to just put it in `src/`. That way we can keep terminal clean with just the terminal logic and the cells and things clearly related to it (please ignore src/term/color.rs, that's in a completely bad spot too).